examples/v1beta1/pipelineruns/6139-regression.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
Can we also have a test that write the result and then exit 1 in the Task ?

Reply:
That example will fail and how can we pass the ci?

Reply:
how do we assert that this yaml test works as expected? 🤔

Reply:
I added an integration test for this case, wonder if that is ok?

Reply:
Yep I think that's ok 👼🏼